mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/Ignore.java

```diff
@@ -20,7 +23,7 @@
  * @since 1.0
  * @deprecated use {@link org.mule.sdk.api.annotation.Ignore} instead.
  */
-@Target(value = {ElementType.FIELD, ElementType.METHOD})
```

Comment:
May be we should only change the mule-sdk-api annotation? If someone needs to Ignore a source, they should use the new annotation.